enterprise/internal/completions/client/llmproxy/llmproxy.go

```diff
@@ -30,7 +30,7 @@ func NewClient(cli httpcli.Doer, endpoint, accessToken string, model string) (ty
 		return openai.NewClient(llmProxyDoer(cli, llmProxyURL, accessToken, "/v1/completions/openai"), "", model), nil
 	}
 	if strings.HasPrefix(strings.ToLower(model), "claude-") {
-		return anthropic.NewClient(llmProxyDoer(cli, llmProxyURL, accessToken, "/v1/completions/anthropic"), "", model), nil
+		return anthropic.NewClient(llmProxyDoer(cli, llmProxyURL, accessToken, "/v1/completions/anthropic"), "", model, endpoint), nil
```

Comment:
To not confuse us that this endpoint might ever be used (the doer overwrites this)
```suggestion
		return anthropic.NewClient(llmProxyDoer(cli, llmProxyURL, accessToken, "/v1/completions/anthropic"), "", model, ""), nil
```